docs/BR.md

```diff
@@ -1739,142 +1739,875 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+| \ \ \ \ `extensions`           | See [Section 7.1.2.4.1](#71241-technically-constrained-tls-subordinate-ca-extensions) |
+| `signatureAlgorithm`           | Encoded value MUST by byte-for-byte identical to the `tbsCertificate.signature`. |
+| `signature`                    | |
+
+##### 7.1.2.4.1 Technically Constrained TLS Subordinate CA Extensions
+
+| __Extension__                     | __Presence__  | __Critical__          | __Description__ |
+| ----                              | -             | -                     | ----- |
+| `authorityKeyIdentifier`          | MUST          | N                     | See [Section 7.1.2.9.2](#71292-authority-key-identifier) |
+| `basicConstraints`                | MUST          | Y                     | See [Section 7.1.2.8.3](#71283-basic-constraints) |
+| `certificatePolicies`             | MUST          | N                     | See [Section 7.1.2.8.4](#71284-certificate-policies---affiliated-ca) (Affiliated CA) |
+| `crlDistributionPoints`           | MUST          | N                     | See [Section 7.1.2.9.3](#71293-crl-distribution-points) |
+| `keyUsage`                        | MUST          | Y                     | See [Section 7.1.2.8.6](#71286-key-usage) |
+| `subjectKeyIdentifier`            | MUST          | N                     | See [Section 7.1.2.9.5](#71295-subject-key-identifier) |
+| `extKeyUsage`                     | MUST[^eku_ca] | -                     | See [Section 7.1.2.8.6](#71286-extended-key-usage---tls-ca) (TLS CA) |
+| `nameConstraints`                 | MUST          | \*[^name_constraints] | See [Section 7.1.2.4.2](#71242-name-constraints) |
+| `authorityInformationAccess`      | SHOULD        | N                     | See [Section 7.1.2.8.2](#71282-authority-information-access) |
+| Signed Certificate Timestamp List | MAY           | N                     | See [Section 7.1.2.9.4](#71294-signed-certificate-timestamp-list) |
+| Any other extension               | SHOULD NOT    | -                     | __**TBD**__ |
+
+##### 7.1.2.4.2 Name Constraints
+
+For a TLS Subordinate CA to be Technically Constrained, Name Constraints extension MUST be encoded as follows. As an explicit exception from RFC 5280, this extension SHOULD be marked critical, but MAY be marked non-critical if compatability with certain legacy applications that do not support Name Constraints is necessary.
+
+Table: `nameConstraints` requirements
+
+| __Field__                 | __Description__ |
+| --                        | -------         |
+| `permittedSubtrees`       | The `permittedSubtrees` MUST contain at least one `GeneralSubtree` for both of the `dNSName` and `iPAddress` `GeneralName` name types, UNLESS the specified `GeneralName` appears within the `excludedSubtrees` to exclude all names of that name type. Additionally, the `permittedSubtrees` MUST contain at least one `GeneralSubtree` of the `directoryName` `GeneralName` naime type. |
```

Comment:
Typo: naime

Reply:
Fixed in https://github.com/sleevi/cabforum-docs/pull/36/commits/b0e0dd1699682b5e5cf9a166d00b1cd1f919f942